bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -236,22 +241,133 @@ for inputstr in smooth_kwargs:
         key, value = inputstr.split(':')
         kwarg_dict[key] = value
     except ValueError:
-            err_txt = "--smoothing-keywords must take input in the " \
-                      "form KWARG1:VALUE1 KWARG2:VALUE2 KWARG3:VALUE3 ... " \
-                      "Received {}".format(' '.join(args.smoothing_keywords))
-            raise ValueError(err_txt)
+        err_txt = "--smoothing-keywords must take input in the " \
+                  "form KWARG1:VALUE1 KWARG2:VALUE2 KWARG3:VALUE3 ... " \
+                  "Received {}".format(' '.join(args.smoothing_keywords))
+        raise ValueError(err_txt)
 
 assert len(args.log_param) == len(args.fit_param) == len(args.smoothing_width)
 
 init_logging(args.verbose)
 
-fits = h5py.File(args.template_fit_file, 'r')
+analysis_time = 0
+attr_dict = {}
+
+# This will be used to check the fit files have the same number of templates
+# (therefore probably using the same bank)
+num_templates = None
```

Comment:
can this be defined closer to where it is used? eg directly before the loop over template fit files.  If so then the second line of comment can probably be removed as it will be visually close enough to the check itself